proto/testing/test.proto

```diff
@@ -3,11 +3,10 @@ syntax = "proto3";
 package testing;
 
 import "google/protobuf/timestamp.proto";
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
 message TestMessage {
   string foo = 1;
-  string bar = 2 [(gogoproto.moretags) = "fizz:\"buzz\" spec-name:\"baz\""];
```

Comment:
idk about this

Reply:
Discussed offline a while ago. It's not a critical feature, your change is sufficient